src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackProcessor.java

```diff
@@ -37,8 +51,35 @@
                                     String jurisdiction,
                                     String caseTypeId,
                                     String caseRef,
-                                    CaseDetails theCase) {
+                                    CaseDetails exceptionRecord) {
+        CcdAuthenticator authenticator = authFactory.createForJurisdiction(jurisdiction);
+        try {
+            CaseDetails theCase = retrieveCase(caseRef, authenticator);
+        } catch (FeignException e) {
+            switch (e.status()) {
+                case 404:
+                    return error("Could not find case: %s", caseRef);
+                default:
+                    return error("Internal Error: Could not retrieve case: %s Error: %s", caseRef, e.status());
+            }
+        }
         return emptyList();
     }
 
+    private CaseDetails retrieveCase(String caseRef, CcdAuthenticator authenticator) {
+        return ccdApi.getCase(authenticator.getUserToken(), authenticator.getServiceToken(), caseRef);
+    }
+
+    private List<String> error(String errorFmt, Object arg) {
+        return error(errorFmt, arg, null);
+    }
+
+    @NotNull
+    private List<String> error(String errorFmt, Object arg1, Object arg2) {
```

Comment:
its better than var-args since its faster ... if I add another I'll move it over to var-args